doc/tutorials/dune-package-management/setup.md

```diff
@@ -0,0 +1,65 @@
+# Setting up Package Management with Dune
+
+The idea of package management with Dune has been to be as unobtrusive as
+possible. Thus most projects can easily be built with just the minimum of
+changes.
+
+In this tutorial we'll create a simple project to use the OCaml package
+management for the very first time.
+
+## Declare dependencies
+
+The best way to work with the package management is to declare your
+dependencies in the `dune-project` file, like so:
+
+```dune
+(lang dune 3.17)
+(name test)
+
+(generate_opam_files true)
+
+(package
+  (name test)
+  (depends
+    (ocaml (>= 4.14))))
+```
+
+## Create lock directory
```

Comment:
```suggestion
## Locking dependencies
```

The fact that its a lock dir is an implementation detail and the intent the user has here is not to create a dir, but to lock the version of the dependencies they need.